manuscript/simulators.tex

```diff
@@ -0,0 +1,27 @@
+\section{Fuel Cycle Simulators}
+
+\subsection{\gls{VISION}}
+\gls{VISION} is a fuel cycle simulation
+that is intended to evaluate "what if" scenarios.
```

Comment:
direct wording from user guide (intended to evaluate 'what if' scenarios) should be avoided.